components/DataSourcesSection/DataSourcesSection.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { FunctionComponent } from 'react';
+import { PageSection, PageSectionHeading } from '../PageSection';
+import { WebsiteUrl } from '../../utils';
+
+export interface DataSourcesLink {
+  caption: string;
+  url: string;
+  page_url: string;
+}
+
+interface DataSourcesSectionProps {
+  description: string;
+  dataSourceLinks?: DataSourcesLink[];
+}
+
+const DataSourcesSection: FunctionComponent<DataSourcesSectionProps> = ({ description, dataSourceLinks }) => {
+  const links = dataSourceLinks?.map((item, index) => {
```

Comment:
Again, things are marked as optional for a reason. You're abusing the power of that `?`... What it means is that you're absolutely sure that dataSourceLinks will NEVER be undefined. Are you? Cos if you are, may as well not make it optional at all. :)